pyiceberg/io/pyarrow.py

```diff
@@ -190,13 +190,6 @@
 T = TypeVar("T")
 
 
-class PyArrowLocalFileSystem(pyarrow.fs.LocalFileSystem):
```

Comment:
People could rely on this for other things, I don't think we can just remove this without deprecation.

Reply:
Let's revert this change, I don't like the inline declaration of the class.